test/scripts/cloudformation-template.coffee

```diff
@@ -0,0 +1,14 @@
+{call, async, merge} = require "fairmont"
+{yaml} = require "panda-serialize"
+
+cloudformation = require "../../src/configuration/cloudformation"
+configuration = require "../../src/configuration"
+
+appRoot = "test/data/test-app/"
+env = "test"
+
+call ->
+  config = yield configuration.read appRoot
+  globals = merge config, {env}
+  cfo = yield cloudformation.renderTemplate appRoot,  globals
```

Comment:
Man this is super-tricky, terminology-wise. I'm so tempted to call this something besides a template. But that's what AWS calls it. On the other hand, that's once it's in YAML/JSON form. Before that, it's not _really_ a template. We're not _really_ rendering it until we turn it into YAML.